frontend/src/views/Collection/utils.ts

```diff
@@ -105,7 +106,7 @@ function buildDoiMetadataLink(
   }
 
   // If there's no summary citation for the collection, return the DOI link with the label "DOI".
-  if (!summaryCitation) {
+  if (summaryCitation === CATEGORY_VALUE_KEY.NO_PUBLICATION) {
```

Comment:
"No value" value is now `NO_PUBLICATION`. [See comment above](https://github.com/chanzuckerberg/single-cell-data-portal/pull/5678/files#r1312136599).